config/SZBE69/splits.txt

```diff
@@ -17,7 +17,7 @@ rb3/main.cpp:
 	.text       start:0x8000F96C end:0x8000F9B0
 
 rb3/string.cpp:
```

Comment:
this is likely a part of some library

i'd put it in `unknown` instead of `rb3` for now